tools/apigee-sackmesser/README.md

```diff
@@ -14,36 +14,57 @@ PATH="$PATH:$PWD/bin"
 To use it as a Docker container you can build the image:
 
 ```sh
-build.sh -t apigee-sackmesser
+./build.sh -t apigee-sackmesser
 ```
 
-## Deploy Command
+## General Usage
 
 ```sh
-$ sackmesser deploy -h
+$ sackmesser --help
 
-usage: sackmesser deploy -e ENV -o ORG [--googleapi | --apigeeapi] [-t TOKEN | -u USER -p PASSWORD] [options]
+usage: sackmesser COMMAND -e ENV -o ORG [--googleapi | --apigeeapi] [-t TOKEN | -u USER -p PASSWORD] [options]
 
-Apigee deployment utility
+Apigee Sackmesser utility.
+
+Commands:
+deploy
+list
+export
 
 Options:
 --googleapi (default), use apigee.googleapi.com (for X, hybrid)
 --apigeeapi, use api.enterprise.apigee.com (for Edge)
 -b,--base-path, overrides the default base path for the API proxy
--d,--directory, path to the apiproxy folder to be deployed
+-d,--directory, path to the apiproxy or shared flow bundle to be deployed
 -e,--environment, Apigee environment name
--g,--github, Link to proxy bundle on github
--n,--api, Overrides the default API proxy name
+-g,--github, Link to proxy or shared flow bundle on github
+-n,--name, Overrides the default API proxy or shared flow name
```

Comment:
breaking change - this will affect openlegacy, will any other projects be affected?

Reply:
(i will change the OL script)